backend/src/@types/knex.d.ts

```diff
@@ -1304,5 +1320,25 @@ declare module "knex/types/tables" {
       TKeyValueStoreInsert,
       TKeyValueStoreUpdate
     >;
+    [TableName.IdentityGroups]: KnexOriginal.CompositeTableType<
```

Comment:
IGNORE PLACEMENT OF COMMENT

Looks like it's missing the identity group org permissions?
<img width="1512" height="842" alt="image" src="https://github.com/user-attachments/assets/eecca9b7-ae78-4e9a-a41c-b4a932c99970" />

Same for the identity group project permissions
<img width="1512" height="839" alt="image" src="https://github.com/user-attachments/assets/0803aec2-d93a-41bf-a4a5-cbd93142ab84" />